tox.ini

```diff
@@ -52,18 +52,17 @@ envlist =
     {py3.8,py3.11}-beam-latest
 
     # Boto3
-    {py2.7,py3.6,py3.7}-boto3-v{1.12}
+    {py3.6,py3.7}-boto3-v{1.12}
     {py3.7,py3.11,py3.12}-boto3-v{1.21}
     {py3.7,py3.11,py3.12}-boto3-v{1.29}
     {py3.7,py3.11,py3.12}-boto3-latest
 
     # Bottle
-    {py2.7,py3.5,py3.9}-bottle-v{0.12}
-    {py3.5,py3.11,py3.12}-bottle-latest
+    {py3.9}-bottle-v{0.12}
```

Comment:
Do we need py 3.6 here (or in any of the other tests where py3.5 was removed)?

Reply:
This is a very good point, adding this!